src/components/AsyncLoad.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import Loadable from 'react-loadable';
+import retry from '../util/retry';
+
+type Props = {
+    loader: () => Promise<any>,
+};
+
+const AsyncLoad = (props: Props) =>
+    Loadable({
+        loading: () => null,
+        ...props,
```

Comment:
passed in props could override loading but not the loader method, is that intended?

Reply:
yeah I want to enforce retries